lib/createTopLevelExpect.js

```diff
@@ -1368,7 +1373,7 @@ expectPrototype._createWrappedExpect = function(
       ((argRule && argRule.type.is('assertion')) ||
         wrappedExpect._getAssertionIndices().indexOf(i) >= 0)
     ) {
-      return new AssertionString(arg);
+      return new AssertionString(utils.forwardFlags(arg, flags));
```

Comment:
Just to confirm, this means we do the substitution using the forwarded flags and then move to process the assertion?

Reply:
This part only has to do with formatting the error message. If we don't forward the flags here, it would come out as:

```js
expected { quux: 123 } to have a foo property of bar

{
  quux: 123
  // missing: foo: expected: noop [not] to equal 'bar'
}
```

Instead of:

```
  // missing: foo: expected: noop to equal 'bar'
```